lib/rust/ast/src/generation.rs

```diff
@@ -3,6 +3,7 @@
 #![allow(unused_must_use)]
 
 use itertools::Itertools;
+use proc_macro2::Span;
```

Comment:
Please fix all the lints that CI is showing in this file. You will need to view it in the files tab to see it.